### PR TITLE
Fix typos in the "Prerequisites for Contributing" page

### PR DESCRIPTION
**Pull request by phildremi**
Separate PR since it requires renaming the entire page 😱

### CODE REVIEW

File: configuration/structure.json [hunk 1]
```diff
@@ -2501,10 +2501,10 @@
                     "friendlyName": "Contribute to Babylon.js",
                     "content": "contribute/toBabylon",
                     "children": {
-                        "Prerequisits": {
-                            "friendlyName": "Prerequisits for Contributing",
+                        "Prerequisites": {
```
Comment (Popov72) [inline configuration/structure.json:2504]: I think we can't make this change, else it will break existing links pointing to this page.

cc @RaananW

Comment (phildremi) [reply configuration/structure.json:2504]: It seems that there are no other references to this page inside the documentation:

image

Nor in the forums (if that's what you were thinking about):

image

Even Google just has this one result:

image

It does seem there's a `sitemap.xml` that I missed (will fix in a separate commit), but nothing else on GitHub either:

image

Comment (Popov72) [reply configuration/structure.json:2504]: I was not thinking of references inside the documentation but references that could have been used/shared by users, either as bookmarks or inside some docs for eg (or sent by emails, ...).

Comment (phildremi) [reply configuration/structure.json:2504]: Gotcha. Though I doubt anyone would bookmark [this minimalistic page](https://doc.babylonjs.com/contribute/toBabylon/Prerequisits), it's of course your decision :)

Perhaps it's possible to set up an alias somehow so that bookmark visitors will be redirected?

Comment (RaananW) [reply configuration/structure.json:2504]: There is a redirection json file that you can use to redirect from the old URL to the new one. Would be great if you add it there!
it will require an approval from an admin (because it is a configuration json), but that shouldn't be a problem.

File: public/sitemap.xml [hunk 2]
```diff
@@ -107,7 +107,7 @@
 <url><loc>https://doc.babylonjs.com/contribute/contributeToDocs/easydoc</loc><lastmod>2022-12-13T12:45:57.562Z</lastmod></url>
 <url><loc>https://doc.babylonjs.com/contribute/contributeToDocs</loc><lastmod>2022-10-17T11:20:30.293Z</lastmod></url>
 <url><loc>https://doc.babylonjs.com/contribute/toBabylon/HowToContribute</loc><lastmod>2022-12-13T12:45:57.563Z</lastmod></url>
-<url><loc>https://doc.babylonjs.com/contribute/toBabylon/Prerequisits</loc><lastmod>2022-09-22T12:30:30.282Z</lastmod></url>
+<url><loc>https://doc.babylonjs.com/contribute/toBabylon/Prerequisites</loc><lastmod>2022-09-22T12:30:30.282Z</lastmod></url>
```
Comment (RaananW) [inline public/sitemap.xml]: sitemap is updated on a weekly base in an external process. You can revert this change

**Review by RaananW — COMMENTED** [review]
Looks great! A redirect is missing, sitemap needs to be reverted.

Thanks a lot!

File: public/sitemap.xml (hunk 2 above)
Comment (phildremi) [reply public/sitemap.xml]: Done. BTW: Is the sitemap update process documented anywhere? I can't seem to find any info about it in the docs.

I'm just curious, but I guess it might be worth documenting if it's something that contributors would need to know about.

File: configuration/structure.json (hunk 1 above)
Comment (phildremi) [reply configuration/structure.json:2504]: I think I found the right file, but I guess the redirects don't work locally. so a deployment on Vercel is the only way to test?

Comment (RaananW) [reply configuration/structure.json:2504]: yes, these are server-side redirects. LGTM! thanks a lot

**Review by RaananW — APPROVED** [review]

File: public/sitemap.xml (hunk 2 above)
Comment (RaananW) [reply public/sitemap.xml]: it's not documented intentionally :-)

But running locally `npm run dev:local` will generate the sitemap
